benchkit/campaign.py

```diff
@@ -402,13 +405,23 @@ def generate_graph(
             else:
                 suite_path = parentdir(suite_path_tentative)
 
-        generate_chart_from_multiple_csvs(
-            csv_pathnames=self.result_csv_paths,
-            plot_name=plot_name,
-            output_dir=suite_path,
-            process_dataframe=process_dataframe,
-            **kwargs,
-        )
+        if use_json:
+            json_files = self.get_json_files(campaign_paths[0])
```

Comment:
this might be a problem when called on a suite. Could we go in all the campaign_paths instead of only the first one?